common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -363,4 +363,39 @@ public boolean useOldFetchProtocol() {
     return conf.getBoolean("spark.shuffle.useOldFetchProtocol", false);
   }
 
+  /**
+   * Class name of the implementation of MergedShuffleFileManager that merges the blocks
+   * pushed to it when push-based shuffle is enabled. By default, push-based shuffle is disabled at
+   * a cluster level because this configuration is set to
+   * 'org.apache.spark.network.shuffle.ExternalBlockHandler$NoOpMergedShuffleFileManager'.
+   * To turn on push-based shuffle at a cluster level, set the configuration to
+   * 'org.apache.spark.network.shuffle.RemoteBlockPushResolver'.
+   */
+  public String mergedShuffleFileManagerImpl() {
+    return conf.get("spark.shuffle.push.based.mergedShuffleFileManagerImpl",
```

Comment:
I liked the suggestion from @tgravescs here in this [comment](https://github.com/apache/spark/pull/30164#discussion_r515289567) which is to use `spark.shuffle.push.X` for push based shuffle introduced configs.

Reply:
I have renamed this configuration to `spark.shuffle.server.mergedShuffleFileManagerImpl`. All the server side configs are prefixed with `spark.shuffle.server` and the names do indicate if they are used for merging shuffle blocks.